pkg/controller/gatewayapi/gatewayapi_controller.go

```diff
@@ -137,18 +178,28 @@ func (r *ReconcileGatewayAPI) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
-	pullSecrets, err := utils.GetNetworkingPullSecrets(installation, r.client)
-	if err != nil {
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Error retrieving pull secrets", err, reqLogger)
-		return reconcile.Result{}, err
+	if variant == operatorv1.Calico {
+		reqLogger.Info("Variant is Calico")
```

Comment:
This info log is going to be too spammy - we should remove it or drop its verbosity to V(2) or higher.

Reply:
Although I am still counting on removing this branch entirely. None of the fields checked by this branch should be enterprise only.